docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -102,6 +102,7 @@ There are the following types:
 * [`TransferSnsTreasuryFunds`](#transfersnstreasuryfunds).
 * [`UpgradeSnsControlledCanister`](#upgradesnscontrolledcanister).
 * [`ManageSnsMetadata`](#managesnsmetadata).
+* [`MintSnsTokens`](#mintsnstokens).
 
 All of the proposals used to manage an SNS are executed on the SNS governance canister so it helps to have for reference, what the [interface for the governance canister](https://sourcegraph.com/github.com/dfinity/ic@4732d8281404c0a7c1e0a91937ffd0e54f2beced/-/blob/rs/sns/governance/canister/governance.did) is.
 
```

Comment:
@anchpop 
1. does the candid example below also have to be updated?
2. I would probably put the description of the critical proposals here.

Reply:
1. You're right, it does
2. let me know if I moved it to the right place!